e2e/playwright/support/kv.ts

```diff
@@ -0,0 +1,64 @@
+import {Client} from 'pg';
+
+export function getClient() {
+    const client = new Client({
+        host: 'localhost',
+        user: 'mmuser',
+        password: 'mostest',
+        database: 'mattermost_test',
+    });
+
+    client.connect();
+    client.end = client.end.bind(client);
+    return client;
+}
+
+export const getKV = async <T>(key: string, pluginId: string): Promise<T | null> => {
```

Comment:
aren't getKV and setKV unused? can they be removed?

Reply:
Sure, YAGNI 🙂